src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -249,7 +249,8 @@
 
             <!-- is this a Staff Only filing? -->
             <template v-if="filing.isTypeStaff">
-              <StaffFiling :filing="filing" class="mt-6" />
+              <StaffFiling :filing="filing" class="mt-6"
+              @downloadOne="downloadOne(...arguments)"/>
```

Comment:
Our usual formatting is to indent the second line and move the closing tag to a new line at the same indent level as the opening tag.

```
   <MyComponent
      line2
      line3
   />
```

Reply:
```
   <MyComponent
      line1
      line2
   >
      <MyInnerComponent />
   </MyComponent>
```